base_geoengine/__openerp__.py

```diff
@@ -3,7 +3,7 @@
 # © 2016 Yannick Vaucher (Camptocamp SA)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {'name': 'Geospatial support for Odoo',
- 'version': '9.0.1.2.5',
+ 'version': '9.0.1.3.6',
```

Comment:
Why this random version? Just bump the next one, not one minor + one revision more